### DISCUSSION TITLE
services failed and stalled

### DISCUSSION THREAD

**Discussion by ninjastrikers** [hiddify/Hiddify-Manager#4595] [Bug باگ: please use issue]
hiddify-panel is failed and other services are stalling at activating state. (after updating and new installing)
![image](https://github.com/user-attachments/assets/3d308ac7-d218-47cf-bfe6-f2d503451ea4)
![image](https://github.com/user-attachments/assets/36ce098b-f983-49ac-95c9-25254bd63960)
![image](https://github.com/user-attachments/assets/8a109dcf-bb36-4555-a408-55ef10b61e94)

**Comment by alireza-abdollahzadeh** [comment]
Hi, I had the same issue this morning. The ha-proxy, panel, and cli services were stuck in "activating" or "failed" states, and restarting or reinstalling the stable version didn’t resolve the problem.
I deleted all the services, updated the server to the beta version, and resolved the issue.
